lstchain/statistics.py

```diff
@@ -66,3 +66,32 @@ def var(self):
     def std(self):
         '''Get the current sample std. dev. of all tracked statistics'''
         return np.sqrt(self.var)
+
+
+
+def sigma_clipped_mean_std(values, axis=0, max_sigma=4, n_iterations=5):
```

Comment:
Hi @maxnoe, at the least the max_sigma should be configurable in the pedestal and flatfield components

Reply:
also the number of iterations

Reply:
Done